andela/travel_tool_front
# #161584429 Fix reconfirmation dialog for delete travel member

#### What does this PR do?
This PR fixes the responsiveness issue with the re-confirmation dialogue box when deleting a travel team member 

#### Description of Task to be completed?
- Add media queries to make the dialogue box responsive

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- Install dependencies and start the  app - `make start`
-  [Start](https://github.com/andela/travel_tool_back/pull/139)  the backend.
- Navigate to [travela](http://travela-local.andela.com:3000) in your browser and login
- Go to the `DB` and on the `UserRoles` table, set yourself as a `Travel Admin`
- Go to the browser, refresh the page, Navigate to the `Settings` tab and  click on `Travel team`
- You should view travel team members in the table shown.
- If none exists add one using the `Add user` button. Make sure he is a registered user in the application.
- Click on the `delete` button in the user row.
- A confirmation dialogue box should appear
- Resize the browser width to different sizes to test responsiveness


#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161584429](https://www.pivotaltracker.com/story/show/161584429)

#### Screenshots (if appropriate)
<img width="1201" alt="screen shot 2018-10-31 at 10 02 06" src="https://user-images.githubusercontent.com/12511966/47771379-0f340b00-dcf4-11e8-8ab0-36d08b5db6c0.png">


#### Questions:
N/A
